context.go

```diff
@@ -967,6 +997,9 @@ func (c *Context) Pow(d, x, y *Decimal) (Condition, error) {
 
 // Quantize adjusts and rounds v as necessary so it is represented with
 // exponent exp and stores the result in d.
+// TODO(mjibson): "adjusts"?
+// TODO(mjibson): Could use a comment (not just here), or a wrapper Function
+// since you call this sometimes with the same argument for both d and v and that scares me.
```

Comment:
I'm not at all concerned about correctness because the testing throughout is ridiculously good, but it is weird to see, but that's fine. I'm OK with it.